### DISCUSSION TITLE
Bonsai Real Sense Device

### DISCUSSION THREAD

**Discussion by Michael-132-wq** [bonsai-rx/bonsai#2052] [Q&A]
Hello, 

I am attempting to utilize a real sense device in my bonsai layout. I would like to use the real sense camera just purely as a color input stream. I have found the color stream option but am having trouble with the Real sense device input module.

Is their documentation on how to use the RealSenseDevice input module?

Thank you,
Michael

**Comment by bruno-f-cruz** [comment]
Hi,
Would you mind sharing your workflow and the link to the source code of the package you are using to connect to the sensor?
Thanks
